spec/classes/core/fiberspec_spec.rb

```diff
@@ -8,6 +8,7 @@
       let(:facts) { os_facts }
 
       it { is_expected.to compile.with_all_deps }
+      it { is_expected.to contain_class('profile::ts::dco') }
```

Comment:
It looks like there are more examples of testing the catalog for a `profile::...` class than I thought there were.  Generally, within a puppet module the goal is test the the resources that end up in catalog and not being sensitive to the internal construction of the module (classes, defines, etc.). The same rule applies for the profiles within our site module.  There are exceptions to the general rule but this case probably shouldn't be an exception.  

Note that as the `fiberspec` example group is being include 2 lines down so this spec test would also be a duplicate.